cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -149,7 +151,7 @@ func generateInitramfsMounts() (err error) {
 	case "recover":
 		err = generateMountsModeRecover(mst)
 	case "install":
-		err = generateMountsModeInstall(mst)
+		err = installNewSystemGenerateMounts(mst)
```

Comment:
is the idea to always do this? should we do this only if preseed is setup?

Reply:
it seems we can't do this in general because of the different hook contract ?

Reply:
I have a rather radical opinion here :)
I thin we should completely eliminate "install boot" for all the scenarios unless proven otherwise that this is needed.
initramfs is exactly what we want as an ephemeral system to run install in.

Reply:
as we discussed we have to:
* start smaller and decide how to trigger this (which probably relates to the question about the hook contract)
* we still have use cases for the full install mode (dedicated installers for example)